functions/public/Update-omsAgent.ps1

```diff
@@ -0,0 +1,179 @@
+#Requires -Version 5.0
+
+function Update-OmsAgent
+{
+	<#
+		.Synopsis
+			Update the OMS agent on remote computers.
+		.DESCRIPTION
+			Either downloads the installer from a URL or copies the installer via the powershell session. Can detected if a previous version is installed and skip if so. If allready installed WorkSpaceId and WorkSpaceKey added to previous install. Doesn't detect invalid workspace IDs or Keys.
```

Comment:
Can the description be updated so it is easier to understand exactly what the CMDLet does.